_posts/2015-10-27-drupal-web-mapping.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "Drupalized Web Mapping @ NACIS 2015"
+date: 2015-10-27
+author: Tim Stallmann
+tags: Drupal, Drupal Planet, Cartography
+summary: Slides from Tim's talk on building web maps using Drupal at the 2015 NACIS conference.
+---
+
+Two weeks ago, I gave a presentation on "[Drupalized Web Mapping](http://timstallmann.github.io/nacis-drupal-mapping-talk/#/)" for the [North American Cartographic Information Society 2015](http://www.nacis.org) conference in Minneapolis, MN.
```

Comment:
purely stylistic, consider removing the quotes around it since it's already a link?